build.sh

```diff
@@ -922,6 +909,21 @@ if [ $__CrossBuild == 1 ]; then
     fi
 fi
 
+if [[ -n "$__NumProc" ]]; then
+    NumProc="$__NumProc"
```

Comment:
I am sorry for not being clear, this is not what I meant. The main goal was to use `__NumProc` everywhere. I don't see a reason for having two variables for the same thing. Also, if you moved this code before the loop parsing the arguments (which is what I meant by putting it to the place where the other option variables are initialized), then you don't need the `if [[ -n "$__NumProc" ]]; then` here - the __NumProc would be set to the extracted number of processors by default and the command line option would just override the value.